example/provider-extensions/ssh-reverse-tunnel/base/ssh/ssh_deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
             command:
             - sh
             - "-c"
-            - "netstat | grep 6222"
```

Comment:
I am not sure if we still have the behaviour with this change as `apk` might as well use port `443`. Hence, the liveness probe might not correctly identify a hanging `apk` process.